awx/ui_next/src/components/CodeMirrorInput/VariablesDetail.jsx

```diff
@@ -9,7 +9,7 @@ import { yamlToJson, jsonToYaml, isJson } from '../../util/yaml';
 const YAML_MODE = 'yaml';
 const JSON_MODE = 'javascript';
 
-function VariablesDetail({ value, label, rows }) {
+function VariablesDetail({ value = '---', label, rows }) {
```

Comment:
You will probably need to rebase this commit to get my pr from yesterday as I introduced code that does the same thing your value does above. I just did it here https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/CodeMirrorInput/VariablesDetail.jsx#L61